WebKitBrowser/CMakeLists.txt

```diff
@@ -187,6 +187,10 @@ if(WEBKIT_GLIB_API)
     target_compile_definitions(${PLUGIN_WEBKITBROWSER_IMPLEMENTATION} PRIVATE WEBKIT_GLIB_API)
 endif()
 
+if(PLUGIN_WEBKITBROWSER_USE_EXACT_PATHS)
```

Comment:
Think this is a new build flag correct?
Could you please also add to the build flag declarations with the correct default value? -> https://github.com/rdkcentral/rdkservices/blob/3a38164553179348b99519dd46dfca75e326142b/WebKitBrowser/CMakeLists.txt#L141

Reply:
@MFransen69, updated accordingly.

Reply:
Thanks! Already had noticed and approved the changes